src/middlewares/publicacion.middlewares.js

```diff
@@ -0,0 +1,38 @@
+const Publicacion = async (req, res) => {
+  console.log(jwt.decode(req.cookies.sesion));
+
+  const { userId: owner } = jwt.decode(req.cookies.sesion);
```

Comment:
```suggestion
  const { userId: owner } = jwt.decode(req.cookies.session);
```
Revisa que sea session en vez de sesion, es un typo